inc/lift/RequestHandle.h

```diff
@@ -227,7 +215,7 @@ class RequestHandle
         CURL* curl_handle,
         CurlPool& curl_pool,
         OnCompleteHandler on_complete_handler = nullptr,
-        size_t max_bytes = 0
+        ssize_t max_download_bytes = -1
```

Comment:
doc blocks actually look out of date (on_complete_handler and max_download_bytes)

Reply:
Updated!